### PR TITLE
Upgrade to celery 5

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -1,9 +1,8 @@
 bumpversion==0.5.3
-coverage==4.5.1
+coverage==5.4
 flake8==3.8.3
 pip==18.1
-python-coveralls==2.9.1
-sphinx==1.8.1
+coveralls==3.0.0phinx==1.8.1
```
Comment (Formulka) [inline requirements_dev.txt]: oops

Comment (matllubos) [reply requirements_dev.txt]: díky

File: django_celery_extensions/management/commands/run_celery.py [hunk 2]
```diff
@@ -7,7 +7,8 @@
 
 
 def start_celery(celery_type, celery_settings, extra_arguments, autoreload=False):
-    starter_celery_cmd = 'celery {} -l info -A {} {}'.format(celery_type, celery_settings, extra_arguments)
+    print('celery --app {} {} {}'.format(celery_settings, extra_arguments, celery_type))
```
Comment (zamazaljiri) [inline django_celery_extensions/management/commands/run_celery.py]: Maybe use `self.stdout.write`?

**Review by zamazaljiri — APPROVED** [review]

File: django_celery_extensions/management/commands/run_celery.py (hunk 2 above)
Comment (matllubos) [reply django_celery_extensions/management/commands/run_celery.py]: this should not be here

**Review by rubickcz — APPROVED** [review]

**Pull request merged**